### DISCUSSION TITLE
[2021-09-11] 주간 회의록

### DISCUSSION THREAD

**Discussion by HyejinYang** [team-mokcha/corabal-beta#20] [주간 회의록]
## 논의 내용

- [x] 전체 프로젝트 공간 재정비 & 일정 재조정
  - 2주 동안 기능 최소 1개 이상 개발하기로 결정
  - organization 생성
  - issue 템플릿 생성하여 브랜치 연동
  - issue와 칸반 보드(프로젝트 탭의 frontend, backend) 연동해 일정 및 작업 관리
  - Wiki 페이지 활용해 기술 문서 아카이빙하기로 결정(페이지 추가 예정)
 
- [x] DB에 월 단위로 필요한 필드 추가
  - 누적 잔/샷/우유/시럽/크림 수
  - 기록한 일 수
  - 목표를 성공한 일 수
 
- [x] DB에 총 누적 잔 수 추가(0잔 포함)
- [x] 고양이 상태 구별 기능 구현은 보류하기로 결정(DB에 고양이 상태 필드는 일단 넣어놓은 채로 있기)
- [x] 개인정보 처리 방침, 이용 약관은 기능 개발이 완료된 후 논의하기로 결정

- [x] 월, 수, 금 오전 9시에 각자 상황 공유하기 (13일 월요일은 오후 10시 예정)

## 진행 사항
다음 주부터 적자.. 다들.. 불태웠다............ 고생이 많습니다....... 🔥 

@HyejinYang @alveloper

**Comment by HyejinYang** [comment]
하기 내용 ERD에 반영했습니다. @alveloper 

1. DB에 월 단위로 필요한 필드 추가
 ```
ID: year-month (key)
   {  
       is_succeed_counts : 0,
       is_recorded_counts : 0,
       summed_up_cups: 0,
       summed_up_shots : 0,
       summed_up_milk: 0,
       summed_up_syrup: 0,
       summed_up_cream: 0,       
     }
```
2. DB에 총 누적 잔 수 추가
 ```
ID: user email(key)
total_cups: 0,
```

**Reply by ghost** [reply]
아주... 아주 멋집니다 혜진님! 그리고 정말 고생 많으셨습니다 👍
